www/routes/index.tsx

```diff
@@ -49,7 +49,12 @@ export default function Home(props: PageProps<Example[]>) {
         <ul class="mt-6 text-gray-900">
           {props.data.map((example) => (
             <li>
-              <a href={`/${example.id}`} class="underline">
+              <a
+                href={`/${example.id}`}
+                class="underline"
+                data-bs-toggle="tooltip"
```

Comment:
This looks like bootstrap specific data attribute. Is this meaningful here? (Or am I missing something?)

Reply:
pretty sure it's how you do tooltips in tailwind

Reply:
I found some usage of `data-bs-toggle` with tailwind such as [Windstrap](https://windstrap.netlify.app/#tooltips) or [Tailwind Elements](https://tailwind-elements.com/docs/standard/components/tooltip/), but we don't use either of them

Reply:
I'll change this when I get back from vacation